README.md

```diff
@@ -117,10 +117,15 @@ The public API is:
     - `mtime`
 - `FSTree.prototype.calculatePatch(newTree, isEqual)` calculate a patch against
   `newTree`.  Optionally specify a custom `isEqual` (see Change Calculation).
+- `FSTree.prototype.addEntries(entries, options)` adds entries to an
+  existing tree. Options are the same as for `FSTree.fromEntries`.
```

Comment:
Can we add something to these docs to call out that `addEntries` will overwrite duplicates?